portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,29 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> messageArrayList = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Gracie!");
+    messageArrayList.add("This function uses a Java ArrayList.");
+    messageArrayList.add("This function uses a fetch() command in JavaScript.");
+    messageArrayList.add("This function uses a WebServlet in Java.");
+    response.setContentType("application/json");
+    Gson gson = new Gson();
+    String message_json = gson.toJson(messageArrayList);
```

Comment:
Google Java style guide says variable names should use camelcase. https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names. So 
message_json becomes messageJson.